src/Plane/PlaneFileGlue.cpp

```diff
@@ -128,8 +132,13 @@ PlaneGlue::Read(Plane &plane, KeyValueFileReader &reader)
     plane.weglide_glider_type = 0;
   if (!has_polar_name)
     plane.polar_name.clear();
-  if (!has_dry_mass)
-    plane.dry_mass = plane.reference_mass;
+  if (!has_empty_mass && has_dry_mass) {
+    auto crew_mass_estimate = plane.polar_name.Contains(_T("PAS)")) ? 180. : 90.;
```

Comment:
Another match in a human-readable string. Please don't.

Reply:
Yip, as above.